tests/pages/firefox/notes.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+from pages.regions.download_button import DownloadButton
+
+
+class FirefoxNotesPage(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/releasenotes/'
+
+    _html_locator = (By.CSS_SELECTOR, 'html')
+    _product_logo_locator = (By.CSS_SELECTOR, '.masthead h2 img')
+    _version_locator = (By.CSS_SELECTOR, 'version > h2')
+    _channel_name_locator = (By.CSS_SELECTOR, 'version > h3')
+    _download_button_locator = (By.CSS_SELECTOR, '.download-list > li:first-child a')
+    _download_all_locator = (By.CSS_SELECTOR, '.all-download a')
+
+    @property
+    def version(self):
+        el = self.find_element(*self._version_locator)
+        return el.textContent
+
+    @property
+    def channel(self):
+        el = self.find_element(*self._channel_name_locator)
+        return el.textContent
+
+    @property
+    def gtm_id(self):
+        el = self.find_element(*self._html_locator)
+        return el.get_attribute('data-gtm-page-id')
+
+    @property
+    def download_button_text(self):
```

Comment:
Testing for download button text feels perhaps overly specific for a functional test (if someone changes the copy in the future, does the website break?). Alternatively, you could test that the correct dl button CSS class is applied correctly for each channel perhaps?

If you do add properties like this, I'd suggest adding them to the [download button page region](https://github.com/mozilla/bedrock/blob/master/tests/pages/regions/download_button.py), so other pages can reuse it.

Reply:
This is also something I had to work really hard to get right in #6823. I suppose a unit test would be more appropriate but I don't know how to unit test logic distributed across 3 templates.

Reply:
I would perhaps favor testing for a CSS class over the button text? I know release notes are only in English (currently), but testing for English strings also makes tests hard to run against other locales.

Reply:
The string is literally the thing I want to test. I want to make sure the right product name is on the button.

Reply:
If the string is the absolutely only way you can be sure that the page is showing the right thing then ok. It's usually something to avoid where possible for the reasons above, but if it's the best thing to do here, then go for it :)